### ISSUE TITLE
QSelect option slot scoped values is wrong, if option is disabled

### ISSUE THREAD

**Issue by FragsterAt** [quasarframework/quasar#16567]
### What happened?

I want to "pin" some options in multiple q-select, it works fine with option 'disable' property, but if i want to use `#option` slot, scoped property `selected` is always `false`

![изображение](https://github.com/quasarframework/quasar/assets/20900647/e21e1334-3321-4277-83d7-27f2d26398f3)

### What did you expect to happen?

Right value of `selected` scoped variable - if it is in v-model array - it must be true 

### Reproduction URL

https://codepen.io/FragsterAt/pen/VwgbKgj?editors=101

### How to reproduce?

1. Open reproduction url
2. Open QSelect menu

### Flavour

Quasar CLI with Vite (@quasar/cli | @quasar/app-vite)

### Areas

Components (quasar)

### Platforms/Browsers

Firefox, Chrome

### Quasar info output

```Shell
Operating System - Windows_NT(10.0.19045) - win32/x64
NodeJs - 18.12.0

Global packages
  NPM - 8.19.2
  yarn - 3.2.4
  @quasar/cli - 1.3.2
  @quasar/icongenie - 2.5.4
  cordova - Not installed

Important local packages
  quasar - 2.12.7 -- Build high-performance VueJS user interfaces (SPA, PWA, SSR, Mobile and Desktop) in record time
  @quasar/app-vite - 1.6.2 -- Quasar Framework App CLI with Vite
  @quasar/extras - 1.16.7 -- Quasar Framework fonts, icons and animations
  eslint-plugin-quasar - Not installed
  vue - 3.3.4 -- The progressive JavaScript framework for building modern web UI.
  vue-router - 4.2.5
  pinia - 2.1.6 -- Intuitive, type safe and flexible Store for Vue
  vuex - Not installed
  vite - 2.9.16 -- Native-ESM powered web dev build tool
  eslint - 8.50.0 -- An AST-based pattern checker for JavaScript.
  electron - Not installed
  electron-packager - Not installed
  electron-builder - Not installed
  register-service-worker - 1.7.2 -- Script for registering service worker, with hooks
  @capacitor/core - Not installed
  @capacitor/cli - Not installed
  @capacitor/android - Not installed
  @capacitor/ios - Not installed

Quasar App Extensions
  *None installed*
```

**Comment by pdanpdan** [comment]
Ups, you are right

**Comment by rstoenescu** [comment]
Thank you for reporting!
Fix will be available in Quasar v2.14.1
